src/textual/css/errors.py

```diff
@@ -32,7 +31,7 @@ class StyleValueError(ValueError):
             error is raised.
     """
 
-    def __init__(self, *args, help_text: HelpText | None = None):
+    def __init__(self, *args: object, help_text: HelpText | None = None):
```

Comment:
along the way, I fixed this as well (I figured as long as I'm touching a file might as well fix everything in it so that mypy --strict doesn't complain; as mentioned in the PR comment above, I only chose files for this PR which had few and "obvious" changes)